.envrc

```diff
@@ -152,6 +153,9 @@ info "Activating virtualenv..."
 
 if [ ! -f "${venv_name}/bin/activate" ]; then
     info "You don't seem to have a virtualenv."
+    # If the version doesn't match the contents of .python-version it will fail
+    [[ $(pyenv version) ]] || \
+        die "Please run: make setup-pyenv && python3.6 -m venv ${venv_name}"
```

Comment:
I think if we added some more whitespace, we can make the command pop out more, something like

```
Please run:

	make setup-pyenv && python3.6 -m venv ${venv_name}

```

Reply:
I found a way to do it!